src/browsers/tob/shared/components/ManhattanPlotNew.js

```diff
@@ -0,0 +1,362 @@
+            {title && (
+              <g transform={`translate(${margin.left}, 40)`}>
+                <text style={{ fontSize: 12, textAnchor: 'left' }}>{title}</text>
+              </g>
+            )}
+            {/* <rect width={width} height={height} fill="none" stroke="black" /> */}
+            <g transform={`translate(${margin.left}, ${margin.top})`}>
+              <defs>
+                <clipPath id="clipManhattanPlot">
+                  <rect width={innerWidth} height={innerHeight} fill="none" pointerEvents="all" />
+                </clipPath>
+              </defs>
+              {/* <rect width={innerWidth} height={innerHeight} fill="none" stroke="black" /> */}
+              <g transform={`translate(0, ${innerHeight})`}>
+                <line x2={`${innerWidth}`} stroke="black" />
+
+                {xScaleLocal.ticks(6).map((tick) => (
+                  <g key={tick} transform={`translate(${xScaleLocal(tick)}, 0)`}>
+                    <text style={{ textAnchor: 'middle' }} dy=".71em" y={9}>
+                      {numberWithCommas(tick)}
+                    </text>
+                    <line y2={6} stroke="black" />
+                  </g>
+                ))}
+              </g>
+              <line y2={`${innerHeight}`} stroke="black" />
+              {yScaleLocal.ticks(4).map((tick) => (
+                <g key={tick} transform={`translate(0, ${yScaleLocal(tick)})`}>
+                  <text key={tick} style={{ textAnchor: 'end' }} x={-6} dy=".32em">
+                    {tick}
+                  </text>
+                  <line x2={-3} stroke="black" />
+                  <line
+                    x2={innerWidth}
+                    stroke={yScaleLocal(tick) === innerHeight ? 'none' : 'lightgrey'}
+                  />
+                </g>
+              ))}
+              <g ref={brushRef} />
+              <g clipPath="url(#clipManhattanPlot)">
+                {data.map((d) => (
+                  <React.Fragment key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(d)}`}>
+                    {(accessors.isSelected(d) || accessors.isReference(d)) && (
+                      <line
+                        key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(
+                          d
+                        )}SelectedLine`}
+                        x1={xScaleLocal(accessors.x(d))}
+                        x2={xScaleLocal(accessors.x(d))}
+                        y1={0}
+                        y2={innerHeight}
+                        strokeDasharray={12}
+                        stroke="black"
+                        opacity="0.5"
+                      />
+                    )}
+                    {accessors.isReference(d) ? (
+                      <g
+                        transform={`translate(${xScaleLocal(accessors.x(d))}, ${yScaleLocal(
+                          accessors.y(d)
+                        )}),rotate(45)`}
+                      >
+                        <rect
```

Comment:
As the specified in the [spec](https://github.com/populationgenomics/exome-results-browsers/issues/70), please use `TooltipAnchor` from `@gnomad/ui` to render tooltips to reduce boilerplate code.

Example usage:
```jsx
<TooltipAnchor key={`${accessors?.id(d) || index}-tooltip`} tooltipComponent={renderTooltip}>
  <circle
    key={accessors?.id(d) || index}
    cx={xScaleLocal(accessors.x(d))}
    cy={yScaleLocal(accessors.y(d)))}
    r={accessors.isSelected(d) ? 6 : 3}
    fill={accessors.color(d)}
    onClick={onClick}
  />
</TooltipAnchor>
```

```jsx
// defined ealier as a react callback depending on accessors.tooltip
const renderTooltip = ({ d }) => {
  if (accessors?.tooltip(d)) {
    if (React.isValidElement(accessors.tooltip(d))) {
      return accessors.tooltip(d)
    }
    return <div>{accessors.tooltip(d)}</div>
  }
}
```